src/components/shared/CardList.js

```diff
@@ -0,0 +1,64 @@
+import "./CardList.css";
+import "../common.css";
+import card1 from "../../images/card1.png";
+import { formatDate, getDaysAgo } from "../../util/DateCalculator";
+import { useEffect, useState } from "react";
+import { GetFolder } from "../../data-access/api";
+
+function CardListItem({ item }) {
+  let card = item.imageSource;
+  const href = item.url;
+  if (!card) {
+    card = card1;
+  }
+  return (
+    <a
+      href={href}
+      className="card flex flex-col"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <div
+        key={card}
```

Comment:
아래와 같이 단락평가를 사용하면 코드를 더 깔끔하게 가져갈 수 있습니다!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Logical_OR#%EB%8B%A8%EB%9D%BD_%ED%8F%89%EA%B0%80

```suggestion
  const href = item.url;

  return (
    <a
      href={href}
      className="card flex flex-col"
      target="_blank"
      rel="noopener noreferrer"
    >
      <div
        key={item.imageSource || card1}
```